src/GeositeFramework/js/Map.js

```diff
@@ -101,20 +100,24 @@ require(['use!Geosite',
         view.model.on('change:extent', function () {
             var currentExtent = view.model.get('extent');
 
-            if (!_.isEqual(currentExtent, view.esriMap.extent)) {
+            if (!_.isEqual(currentExtent, view.esriMapView.extent)) {
                 loadExtent(view);
             }
         });
 
+        /*
         // Configure the esri proxy, for (at least) 2 cases:
         // 1) For WMS "GetCapabilities" requests
         // 2) When it needs to make an HTTP GET with a URL longer than 2000 chars
         esri.config.defaults.io.proxyUrl = "proxy.ashx";
```

Comment:
Looked through the docs some more and it looks like I may be able to un-comment this and adjust how it's set instead:

https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#portalUrl